test/mocks.ts

```diff
@@ -26,4 +26,35 @@ module Mocks {
       return true;
     }
   }
+
+  export class Scale<D, R> extends Plottable.Scale<D, R> {
```

Comment:
The mock is supposed to be extremely simple (Note the previous mock defined above), and if the mock gets too complex then it is not a good mock.

This mock essentially obeys the fundamentals of a what a scale does.  It takes a domain value and then returns a scaled range value.